modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -165,13 +163,7 @@ class PersistentActor[S, M, R, E](
 
   private case class EventPublishTimedOut(reason: Throwable, startTime: Instant) extends Internal
 
-  protected case class InternalSpans(messageProcessingSpan: Option[Span] = None)
```

Comment:
as mentioned in the other PR, this stuff was getting in my way so I got rid of it (for now)